recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
     - jom                   # [win]
     - cmake
     - make                  # [unix]
-    - python                # [aarch64]
+    - python
```

Comment:
This is not ideal...
@isuruf, do you have any other ideas how we could avoid putting all Python builds into the same job?
(If there is no better one, splitting `py-lief` into a separate feedstock would be possible in the end..)